bindings/python/src/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-add_subdirectory(mesh)
-add_subdirectory(model)
+add_subdirectory(mesh_geosciencesio)
```

Comment:
mesh_geosciencesio or geosciencesio_mesh

Reply:
I put it at the end in order to have lib name + project name if lib name is not enough